department-of-veterans-affairs/va.gov-cms
# VACMS-2811 add timezones to date fields

## Description
See #2811 

## Testing done
Visual / Behat

## Screenshots
![image](https://user-images.githubusercontent.com/2404547/95259999-014b6680-07f6-11eb-9dce-e9f2a8b1f911.png)
![image](https://user-images.githubusercontent.com/2404547/95260019-0c05fb80-07f6-11eb-9f43-ab589a7a2471.png)
![image](https://user-images.githubusercontent.com/2404547/95259978-fabcef00-07f5-11eb-95e7-903cb4654029.png)

## QA steps
Visually verify that new smart time field (Date and time) appears, and is disabled on node and view forms for events, news releases, and situation update paragraphs.
